src/main/protobuf/soc/state.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto2";
+
+package protos.soc;
+
+import "soc/board.proto";
+import "soc/inventory.proto";
+
+enum TurnPhase {
+    ROLL = 0;
+    BUY_TRADE_OR_END = 1;
+}
+
+message Player {
+    required string name = 1;
+    required int32 position = 2;
+}
+
+message PublicPlayerState {
+    required Player player = 1;
+    required int32 points = 2;
+    required PublicInventory public_inventory = 3;
+    required int32 longest_roads = 6;
```

Comment:
`longest_road_count` or `longest_road_length`.. I would suggest `knight_count` and `road_count`. Brief and specific.

Reply:
Thats fine. It is a little redundant since that info is already in the played development cards set

Reply:
nevermind I realize that information is already in there